sellotape/sellotape_dj/sellotape_dj/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('auth_app/', include('sellotape_dj.auth_app.urls')),
```

Comment:
I wonder about auth being its own app - shouldn't this be embedded into the actual app that needs authentication?

Reply:
It's only for this stage of development, when we have code in our project this will change.